test-runner-frontend/src/Views/DashboardUtils.js

```diff
@@ -0,0 +1,189 @@
+const HIGHER_IS_BETTER = 'higherIsBetter';
```

Comment:
can we please write unit tests for these functions?

Reply:
Absolutely.